From c1832f4a74177c39f3362b255d9d7127d02813f7 Mon Sep 17 00:00:00 2001
From: Richard Hughes <richard@hughsie.com>
Date: Thu, 25 Jan 2018 13:52:02 +0000
Subject: [PATCH] Do not use the CDN when getting metadata

We have a new server that can hopefully cope with the load.
---
 data/remotes.d/lvfs-testing.conf | 2 +-
 data/remotes.d/lvfs.conf         | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/data/remotes.d/lvfs-testing.conf b/data/remotes.d/lvfs-testing.conf
index 75da08f1..dc4133a5 100644
--- a/data/remotes.d/lvfs-testing.conf
+++ b/data/remotes.d/lvfs-testing.conf
@@ -4,7 +4,7 @@
 Enabled=false
 Title=Linux Vendor Firmware Service (testing)
 Keyring=gpg
-MetadataURI=https://s3.amazonaws.com/lvfsbucket/downloads/firmware-testing.xml.gz
+MetadataURI=https://fwupd.org/downloads/firmware-testing.xml.gz
 ReportURI=https://fwupd.org/lvfs/firmware/report
 Username=
 Password=
diff --git a/data/remotes.d/lvfs.conf b/data/remotes.d/lvfs.conf
index 51d0f55e..b7dea262 100644
--- a/data/remotes.d/lvfs.conf
+++ b/data/remotes.d/lvfs.conf
@@ -4,6 +4,6 @@
 Enabled=true
 Title=Linux Vendor Firmware Service
 Keyring=gpg
-MetadataURI=https://s3.amazonaws.com/lvfsbucket/downloads/firmware.xml.gz
+MetadataURI=https://fwupd.org/downloads/firmware.xml.gz
 ReportURI=https://fwupd.org/lvfs/firmware/report
 OrderBefore=fwupd
-- 
2.14.3

